src/views/ConsentContinuationOut.vue

```diff
@@ -52,34 +52,61 @@
     </v-fade-transition>
 
     <!-- Main Body -->
-    <v-container id="correction-container" class="view-container" v-show="dataLoaded">
-      <v-row>
+    <v-container id="correction-container" class="view-container" v-if="dataLoaded">
+      <v-row no-gutters>
         <v-col cols="12" lg="9">
           <article id="correction-article">
             <!-- Page Title -->
             <header>
               <h1 id="correction-header">Six-Month Consent to Continue Out</h1>
             </header>
 
-            <!-- Detail -->
+            <!-- Ledger Detail -->
             <section>
               <header>
-                <h2 id="correction-step-1-header">1. Detail</h2>
+                <h2>Ledger Detail</h2>
                 <p>Enter a detail that will appear on the ledger for this entity.</p>
-                <p class="black--text mb-0">{{defaultComment}}</p>
               </header>
-              <DetailComment
-                v-model="detailComment"
-                placeholder="Add a Detail that will appear on the ledger for this entity."
-                :maxLength="maxDetailCommentLength"
-                @valid="detailCommentValid=$event"
-              />
+              <v-card flat class="py-8 px-5">
+                <v-row no-gutters>
+                  <v-col cols="12" sm="3" class="pr-4">
+                    <strong>Detail</strong>
+                  </v-col>
+                  <v-col cols="12" sm="9">
+                    <p class="black--text">{{defaultComment}}</p>
```

Comment:
This might not be the text colour that UX wants. It's probably gray9.

Reply:
Checked with Ethan, it's gray7 👍